spec/acceptance/apt_key_provider_spec.rb

```diff
@@ -27,7 +27,8 @@ def populate_default_options_pp(value)
 
 def install_key(key)
   retry_on_error_matching do
-    shell("apt-key adv --keyserver hkps.pool.sks-keyservers.net --recv-keys #{key}")
+    shell("wget -O /tmp/sks-keyservers.netCA.pem https://sks-keyservers.net/sks-keyservers.netCA.pem && apt-key adv --keyserver hkps://hkps.pool.sks-keyservers.net \
+             --keyserver-options ca-cert-file=/tmp/sks-keyservers.netCA.pem  --recv-keys #{key}")
```

Comment:
if you are splitting the command into 2, it should be 2 separate shell commands, so it debugs easier